cpp/src/io/parquet/page_hdr.cu

```diff
@@ -412,6 +403,9 @@ __global__ void __launch_bounds__(128)
         bs->page.lvl_bytes[level_type::DEFINITION] = 0;
         bs->page.lvl_bytes[level_type::REPETITION] = 0;
         if (parse_page_header(bs) && bs->page.compressed_page_size >= 0) {
+          if (not is_supported_encoding(bs->page.encoding)) {
+            error[warp_id] |= static_cast<int>(decode_error::UNSUPPORTED_ENCODING);
```

Comment:
Do these also need to match the type of `int32_t error[4]`? This cast occurs in at least two places that I can see.
```suggestion
            error[warp_id] |= static_cast<int32_t>(decode_error::UNSUPPORTED_ENCODING);
```

Reply:
yep, they sure do. thanks!